spec/services/adapters/structs/crime_application_spec.rb

```diff
@@ -9,7 +9,7 @@
     end
 
     it 'sets the benefit_check_result from the means attribute' do
-      expect(subject.applicant.benefit_check_result).to be true
+      expect(subject.applicant.benefit_check_result).to be false
```

Comment:
The `benefit_check_result` attribute is set to `false` in the fixture which is why this value changes